cinema-soft/src/main/java/softserve/academy/cinemasoft/controller/MovieController.java

```diff
@@ -58,15 +58,20 @@ public String addMovie(Model model) {
 
     @PostMapping("/add-movie")
     public String addMovieFromView(@Valid @ModelAttribute("movie") Movie movie, BindingResult bindingResult,
-                                   @ModelAttribute("fileImage") MultipartFile imageFile, Model model) throws IOException {
+                                   @ModelAttribute("fileImage") MultipartFile imageFile, Model model)  {
 
         if (bindingResult.hasErrors()) {
             model.addAttribute("categories", this.categoryService.findAll());
             MultipartFile file = null;
             model.addAttribute("imageFile", file);
             return "add-movie";
         }
-        byte[] cover = imageFile.getBytes();
+        byte[] cover = new byte[0];
+        try {
+            cover = imageFile.getBytes();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
In this case you should handle the Exception, either by returning a proper error page with information of what went wrong, or a message in the same page.

Reply:
ok